x-pack/plugins/security_solution/server/endpoint/routes/suggestions/index.ts

```diff
@@ -62,6 +63,11 @@ export const getEndpointSuggestionsRequestHandler = (
     let index = '';
 
     if (request.params.suggestion_type === 'eventFilters') {
+      if (!EXCEPTIONABLE_ENDPOINT_EVENT_FIELDS.includes(fieldName)) {
+        return response.badRequest({
+          body: `Unsupported field name: ${fieldName}`,
+        });
+      }
```

Comment:
Is there a way you can use/unify using the `validateField`function that's in [`event_filter_validator`](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/event_filter_validator.ts#L23)

Reply:
Yeah you are right, we could reuse that function. On the other hand, it might imply a bit of refactor on that one since it returns an error string so we should change it to throw instead in order to be consistent. I'm not opposed to reuse it but would avoid having too much changes for that. Reusing it as it is would be something like:

```typescript
if (validateField(fieldName)) {
  return response.badRequest({
    body: `Unsupported field name: ${fieldName}`,
  });
}
```

And, for me, seems less descriptive while reading the code.

Renaming that function to something like `isThisAValidField` and returning a `boolean` would help, but that implies a refactor on the event_filters_validator too.

Thoughts?

Reply:
I see your point. We can leave it as it is now.

Or maybe we can have a function like you suggest with a more descriptive name
```
isExceptionableField(fieldName:string): boolean
``` 
and substitute it wherever we're using `!EXCEPTIONABLE_ENDPOINT_EVENT_FIELDS.includes(fieldName)` as

```
!isExceptionableField(fieldName)
```